### ISSUE TITLE
The module doesn't support the Express editions of Visual Studio

### ISSUE THREAD

**Issue by AnthonyMastrean** [Iristyle/Posh-VsVars#2]
(I'm using Express 2012 web and desktop)

The Express editions install all the correct registry keys for detecting the latest installed version, but they don't have the `InstallDir` property. They do have this other key/value, though...

``` batch
PS> Get-ItemProperty HKLM:\Software\Wow6432Node\Microsoft\VisualStudio\SxS\VS7


11.0         : C:\Program Files (x86)\Microsoft Visual Studio 11.0\
```

Which identifies a more root directory than the regular install, so you'd have to build up the `$VsRootDir` differently...

``` powershell
$VsKey = Get-ItemProperty -Path "$script:rootVsKey\SxS\VS7" -ErrorAction SilentlyContinue
$VsRootDir = Join-Path $VsKey.$version 'Common7'
```

I don't know if you'd always try to search both paths or implement a switch to choose strategies.

``` powershell
[switch] $Express
```

**Comment by shishkin** [comment]
VS Express 2013 for Web doesn't have vsvars32.bat in C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\Tools\ (can't find it anywhere actually).

**Comment by Iristyle** [comment]
Yeah, I think the alternative here might be to just fall back to the default .NET tools path for latest .NET -- this will be something like `C:\Windows\Microsoft.NET\Framework\v4.0.30319`
